build/mixer.sh

```diff
@@ -18,6 +18,7 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 
 var_load_all
 
+MIXER_LOG=${MIXER_LOG:-"mixer-tools.log."$(date "+%Y.%m.%d-%H.%M.%S")}
```

Comment:
On distro-factory, the workdir is always empty in the start of a build and it is used to store files/artifacts relatives to a single build, therefore there is no need to annotate a timestamp to the log, name collision wont be a problem.

I would update this to be just:
MIXER_LOG=${MIXER_LOG:-"${WORK_DIR}/mixer.log"}

Reply:
Done.